### ISSUE TITLE
Corruption in REPL and display issue

### ISSUE THREAD

**Issue by PallHaraldsson** [JunoLab/Juno.jl#613]
I was asked to open here (and in VS Code if applies, but I do not know that for a fact, so I do not for now).

I can't reproduce, and since Juno is EOL I and maybe you shouldn't waste much time on this.

![image](https://user-images.githubusercontent.com/8005416/96453874-9bee7100-120a-11eb-96af-696b66d087da.png)

See more details at: https://github.com/JuliaLang/julia/issues/38082 where I originally filed.

**Comment by PallHaraldsson** [comment]
FYI: It's for sure a bug, and there's a similar one in VS Code, I find it likely to involve memory running out (and/or threads). And there's probably a different bug in Julia, and both triggered by memory running out.

It's probably not a priority for anyone to fix Juno (or more likely Atom), but one question, any idea what it has in common with VS Code? Only Julia and some shared terminal code in Atom and VS Code? And to possibility of memory running out and not being handled well.

**Comment by pfitzseb** [comment]
It is *very* likely this is completely unrelated to Julia and just a bug in xterm.js.

**Comment by PallHaraldsson** [comment]
Yes (and then Atom, if that, not Juno). Is that also used by VS Code? Either way, we can probably close here.

Note, the reorder of REPL output in Julia, is likely a different bug that happens at the same time, more or less. And that would be a Julia bug (already filed), and I'm not sure it's an argument to keep this one open.

**Comment by pfitzseb** [comment]
xterm.js is used both by Juno and VSCode, yes.

The REPL reordering issue might be a bug in Juno, but should've been fixed in VSCode and Revise a while ago.
